hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/exceptions/SCMException.java

```diff
@@ -123,6 +123,7 @@ public ResultCodes getResult() {
     FAILED_TO_INIT_CONTAINER_PLACEMENT_POLICY,
     FAILED_TO_ALLOCATE_ENOUGH_BLOCKS,
     INTERNAL_ERROR,
-    FAILED_TO_INIT_PIPELINE_CHOOSE_POLICY
+    FAILED_TO_INIT_PIPELINE_CHOOSE_POLICY,
+    NOT_SUPPORTED_OPERATION,
```

Comment:
New SCMException introduced here that broken the unit test:

[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.077 s <<< FAILURE! - in org.apache.hadoop.hdds.scm.exceptions.TestSCMExceptionResultCodes
[ERROR] codeMapping(org.apache.hadoop.hdds.scm.exceptions.TestSCMExceptionResultCodes)  Time elapsed: 0.023 s  <<< FAILURE!
java.lang.AssertionError: expected:<30> but was:<29>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at org.apache.hadoop.hdds.scm.exceptions.TestSCMExceptionResultCodes.codeMapping(TestSCMExceptionResultCodes.java:35)

Reply:
yup